.github/workflows/workflow-generate-ci-documentation.yml

```
@@ -0,0 +1,64 @@
+name: Generate CI Documentation
+on:
+  workflow_call:
+    inputs:
+      commit_resources:
+        description: 'Commit the resources after generating them. (Requires the access_token to be passed)'
+        required: false
+        default: false
+        type: boolean
+    secrets:
+      access_token:
+        description: 'The access token to use for commits.'
+        required: false
+  workflow_dispatch:
+    branches:
+    - main
+    - develop
+    - "feature-*"
+    - "release-*"
+    inputs:
+      commit_resources:
+        description: 'commit the resources after generating them. Requires a PAT defined as secrets.COMMIT_TOKEN'
+        required: true
+        default: false
+        type: boolean
+jobs:
+  update_ci_documentation:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Generate CI Documentation
+        uses: Compton-NIST/Inspector@v1
+        with:
+          artifact_name: docs_ci
+          output_path: docs_ci
```

Blocking: can we move this to `build_ci`? I am not sure this applies to the `artifact_name` as well, I will review and examine the source action code and comment accordingly once I understand it, or ask you for clarification. :-)

```suggestion
          output_path: build/ci-cd
```